gap/SkeletalFinSets.gi

```diff
@@ -985,6 +977,27 @@ end, 1 + Sum( [ [ "ExponentialOnObjects", 1 ],
 
 end );
 
+##
+AddDerivationToCAP( IsHomSetInhabited,
+        "IsHomSetInhabited using IsInitial when the range category of the homomorphism structure is the skeletal category of finite sets",
```

Comment:
General comment which does not block the PR: The information you provide in this description is nearly the same as would be displayed by `DerivationsOfMethodByCategory` anyway. Unfortunately, this is the case for many descriptions of derivations. It would be more useful if descriptions would describe the mathematics/semantics behind the derivations.